### PR TITLE
Creating an ELI5 like guide for CMDRs

### PR DESCRIPTION
**Pull request by SayakMukhopadhyay**
A guide doesn't exist for a complete beginner on how to update data on the various websites and this guide is for those CMDRs. Initial idea: https://discord.com/channels/164411426939600896/308627084303925248/828600476298772550

### CODE REVIEW

File: cmdrs-guide.html [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+            When you start Elite Dangerous and do activities in it, the game writes some particular files in the system
+            you are playing on. These files keep track of your activities by writing
+            <a href="https://www.w3schools.com/whatis/whatis_json.asp">JSON</a> data. These files are called the Journal
+            files. By default you can find these files at
+            <code>C:\Users\%userprofile%\Saved Games\Frontier Developments\Elite Dangerous</code> (Please note that
+            depending on your setup, the location might be different).
+        </p>
+        <p>
+            The files in this folder can track back years if you haven't changed your system. That means these files do
+            not carry over when you switch PCs. One way to carry them over is to manually copy from the ols system and
```
Comment (Athanasius) [inline cmdrs-guide.html]: s/ ols / old /

File: cmdrs-guide.html [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        </p>
+        <h4>Elite Dangerous Data Network (EDDN)</h4>
+        <p>
+            Elite Dangerous Data Network or EDDN is a community run message broker used by browser based tools. Client
+            tools send data to EDDN and browser based tools get the data from EDDN. EDDN itself doesn't store any data.
+            It just passes along data.
+        </p>
+        <h4>Commander API (CAPI)</h4>
+        <p>
+            The commander API or the CAPI is an <a href="https://en.wikipedia.org/wiki/API">API</a> provided by the game
```
Comment (Athanasius) [inline cmdrs-guide.html]: 'CAPI' is 'Companion API' - so named from the long-since defunct iOS 'Elite Dangerous Companion' app.

File: cmdrs-guide.html (hunk 1 above)
Comment (SayakMukhopadhyay) [reply cmdrs-guide.html]: I blame my spellchecker 😛

File: cmdrs-guide.html (hunk 2 above)
Comment (SayakMukhopadhyay) [reply cmdrs-guide.html]: 🤦🏼‍♂️

**Comment by SayakMukhopadhyay** [comment]
Please do check if the language or tone could be improved.

File: cmdrs-guide.html [hunk 3]
```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv="X-UA-Compatible" content="chrome=1">
+    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
+    <link rel="stylesheet" type="text/css" href="stylesheets/github-dark.css" media="screen">
+
+    <title>E:D Community Developers - CMDRs Guide</title>
```
Comment (Venefilyn) [inline cmdrs-guide.html:9]: Since 2016 or so `Elite: Dangerous` changed to `Elite Dangerous`. Unrelated to PR since you use the correct style where it is used, but EDCD should change the name to be without colon

File: cmdrs-guide.html [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+        <h3>How do these tools get their data?</h3>
+        <p>
+            There are 3 ways tools get their data:
+        </p>
+        <ol>
+            <li>
+                From journal files generated by Elite Dangerous. Some use the data of these files sent through EDDN.
+            </li>
+            <li>
+                From CAPI once you authenticate your Frontier account with the tools.
```
Comment (Venefilyn) [inline cmdrs-guide.html]:
```suggestion
                From <a href="#capi">CAPI</a> once you authenticate your Frontier account with the tools.
```

File: cmdrs-guide.html [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+            not carry over when you switch PCs. One way to carry them over is to manually copy from the old system and
+            paste them in the new system.
+        </p>
+        <h4>Elite Dangerous Data Network (EDDN)</h4>
+        <p>
+            Elite Dangerous Data Network or EDDN is a community run message broker used by browser based tools. Client
+            tools send data to EDDN and browser based tools get the data from EDDN. EDDN itself doesn't store any data.
+            It just passes along data.
+        </p>
+        <h4>Companion API (CAPI)</h4>
```
Comment (Venefilyn) [inline cmdrs-guide.html]:
```suggestion
        <h4 id="capi">Companion API</h4>
```

File: cmdrs-guide.html [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+        <p>
+            The companion API or the CAPI is an <a href="https://en.wikipedia.org/wiki/API">API</a> provided by the game
```
Comment (Venefilyn) [inline cmdrs-guide.html]:
```suggestion
            The Companion API (CAPI) is an <a href="https://en.wikipedia.org/wiki/API">API</a> provided by the game
```

File: cmdrs-guide.html [hunk 7]
```diff
@@ -0,0 +1,137 @@
...
+            depending on your setup, the location might be different).
+        </p>
+        <p>
+            The files in this folder can track back years if you haven't changed your system. That means these files do
+            not carry over when you switch PCs. One way to carry them over is to manually copy from the old system and
+            paste them in the new system.
+        </p>
+        <h4>Elite Dangerous Data Network (EDDN)</h4>
+        <p>
+            Elite Dangerous Data Network or EDDN is a community run message broker used by browser based tools. Client
```
Comment (Venefilyn) [inline cmdrs-guide.html]:
```suggestion
        <h4>Elite Dangerous Data Network</h4>
        <p>
            Elite Dangerous Data Network (EDDN) is a community run message broker used by browser based tools. Client
```

File: cmdrs-guide.html [hunk 8]
```diff
@@ -0,0 +1,137 @@
...
+            <li>
+                From manual user input
+            </li>
+        </ol>
+        <h3>Whoa! What are journal files, EDDN and CAPI?</h3>
+        <h4>Journal Files</h4>
+        <p>
+            When you start Elite Dangerous and do activities in it, the game writes some particular files in the system
+            you are playing on. These files keep track of your activities by writing
+            <a href="https://www.w3schools.com/whatis/whatis_json.asp">JSON</a> data. These files are called the Journal
```
Comment (Venefilyn) [inline cmdrs-guide.html:88]:
```suggestion
            <a href="https://en.wikipedia.org/wiki/JSON">JSON</a> data. These files are called the Journal
```

File: cmdrs-guide.html [hunk 9]
```diff
@@ -0,0 +1,137 @@
...
+        </p>
+        <p>
+            Please keep in mind that these tools are created and maintained by developers for free in their free time at
+            a best effort basis. Some of these developers can be supported via various channels. If possible, please
+            support their efforts. Links would be available in the respective locations of the tools.
+        </p>
+        <h3>What are these tools and where can I find them?</h3>
+        <p>
+            You can find links to some of these tools in the <a href="index.html">homepage</a>. You can find more at
+            <a href="http://edcodex.info/">Elite:Dangerous Codex</a>. There are a vast number of tools, big or small
```
Comment (Venefilyn) [inline cmdrs-guide.html]:
```suggestion
            <a href="http://edcodex.info/">Elite:Dangerous Codex</a>. There are a vast number of tools
```

**Review by Venefilyn — COMMENTED** [review]
Won't go into the writing, but a lot of it could be reworded to flow better and be less quirky (like "Okay!" in headers etc)

File: cmdrs-guide.html (hunk 3 above)
Comment (SayakMukhopadhyay) [reply cmdrs-guide.html:9]: Yep. I kept the current style but I agree with you.

File: cmdrs-guide.html (hunk 8 above)
Comment (SayakMukhopadhyay) [reply cmdrs-guide.html:88]: I used the w3school link since it explains things from a learners perspective.

**Comment by SayakMukhopadhyay** [comment]
> 
> 
> Won't go into the writing, but a lot of it could be reworded to flow better and be less quirky (like "Okay!" in headers etc)

I have removed the unnecessary exclamations in the headers. Can you give some idea as to what can be improved to flow better?

**Comment by Venefilyn** [comment]
> I have removed the unnecessary exclamations in the headers. Can you give some idea as to what can be improved to flow better?

It all comes down to UX writing, you can have quirks but in most cases it doesn't really work well. Focusing on writing like a story works well when addressing people wanting to know more, which is what this page aims to do

Just going to describe some of the paragraphs

> Elite Dangerous is a vast and complex game and there are many different moving parts in the game. Although in-game tools have improved over the years, it still becomes very difficult to progress in the game by only using them. As such, many players of the game have created their own tools available outside the game which greatly enhances the ease of playing Elite Dangerous. These tools are not the official tools of Elite Dangerous and their developers have no affiliation to Frontier Developments.

Good example of good flow. It goes from what the game is, difficulties that exist, that players created their own tools. It reads like a story, which makes it easier to consume. Last sentence about being unaffiliated does not work well though with that story, it might even be unneeded as we are mentioning people creating their own tools.

---

But when there's descriptions of tools or details about a topic, most of the sentences on the page are quite dry. Not something you'd be able to tell someone verbatim without seeming weird.

Take the first occurrence of this:

> ### What are these tools and where can I find them?
> You can find links to some of these tools in the [homepage](#). You can find more at [Elite:Dangerous Codex](http://edcodex.info/). There are a vast number of tools that you might discover but these should be enough to get you started.

We could rewrite this similar to the story in the beginning

> ### What are these tools and where can I find them?
> We will go over a few of the most important tools on this page, but there are a lot of tools out there we won't cover. On the [homepage](#) we list a few more worth mentioning, there is also [Elite:Dangerous Codex](http://edcodex.info/) that contains a vast list of tools people have created.

Though not necessary, it helps create a better flow IMO. And addressing the other paragraphs in the same way would help a lot

---

Taking another paragraph that does have a good story:

> Most tools running on your computer will read the [journal files](#). Some of these tools will use the data from the journal to show you output in the tool itself. Most of these tools will also have an option to send the data to EDDN. Please note that EDDN doesn't accept all data that is generated from the journal files to protect the users privacy.

I would say it is good as it is now, but it reads "most tools will read x. some tools take info from x to display. most tools can send [x] to y." which to me has very sharp breaks (don't know how to explain it better, I'm not very good with UX writing). You may after first sentence get a question of "why do they read journal files" only to be answered by the next paragraph. In my opinion, it should either hint that it will be explained (better for larger or complicated topics), or answer at least parts of it in that sentence.

Instead you could rephrase it "most tools reading x can display info and may also send it to EDDN".

> Most tools running on your computer will read the [journal files](#) to display info in the tool itself or send it to other services, like EDDN.

The last sentence can be dropped in this case IMO as it doesn't matter to the question of how they all fit together. If they want to read more on EDDN they can do so under the right header and it will be explained there of what precautions EDDN takes

File: cmdrs-guide.html [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+            depending on your setup, the location might be different).
+        </p>
+        <p>
+            The files in this folder can track back years if you haven't changed your system. That means these files do
+            not carry over when you switch PCs. One way to carry them over is to manually copy from the old system and
+            paste them in the new system.
+        </p>
+        <h4 id="eddn">Elite Dangerous Data Network (EDDN)</h4>
+        <p>
+            Elite Dangerous Data Network (EDDN) is a community run message broker used by browser based tools. Client
```
Comment (Venefilyn) [inline cmdrs-guide.html:100]: Reason I suggested to remove abbreviation in the header and add it to the body is just to align with what I would expect from docs. But I wouldn't have it explained twice

If you want abbreviation in the header it would be better to have the paragraph starts with `EDDN is...` since the abbreviation will be used the most throughout the page

Comment (SayakMukhopadhyay) [reply cmdrs-guide.html:100]: I agree that might work. But I wanted to make this doc idiot proof and hence explaining stuff more than once if necessary. Still if the community feels strongly about this, I will remove it.

**Comment by SayakMukhopadhyay** [comment]
> Last sentence about being unaffiliated does not work well though with that story, it might even be unneeded as we are mentioning people creating their own tools.

The reason I added that is because many newcomers come to the EDCD server thinking that the tools are made by FDev and they start expecting FDev like support. So, I am clearing it up as early as possible.

**Comment by SayakMukhopadhyay** [comment]
I will wait for a mantainer to respond to this PR before deciding to do any more modifications.

File: cmdrs-guide.html (hunk 3 above)
Comment (Athanasius) [reply cmdrs-guide.html:9]: I'm all for a global E:D -> ED replace.  The only reason I've not done it in EDMC text is related to translations and our documentation also needing updating to match.

**Review by AnthorNet — APPROVED** [review]
